### PR TITLE
clarify logo-uri

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Improve description for logo-uri, also fix table (lots of pointless line spaces)
- Changed "screen" to "dialog"

### Resolves:

* [OKTA-133644](https://oktainc.atlassian.net/browse/OKTA-133644)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@jakubvul Please review this change. Especially, any reason not to say "dialog" instead of screen or window? Thank you!

**Comment by cwu-okta** [comment]
@federations-okta

File: _source/_docs/api/resources/oauth-clients.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Client applications have the following properties:
...
 | client_secret               | OAuth 2.0 client secret string (used for confidential clients)                                          | String                                                                                       | TRUE     | TRUE   | TRUE      |
 | client_secret_expires_at    | Time at which the client_secret will expire or 0 if it will not expire(measured in unix seconds)        | Number                                                                                       | TRUE     | FALSE  | TRUE      |
-| logo_uri                    | URL string that references a logo for the client                                                        | String                                                                                       | TRUE     | FALSE  | FALSE     |
+| logo_uri                    | URL string that references a logo for the client approval dialogs (not sign-in dialogs)                 | String                                                                                       | TRUE     | FALSE  | FALSE     |
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oauth-clients.md]: I guess the approval dialogs are the consent dialogs? Should we use the term approval or consent?

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth-clients.md]: Let's use consent, I'll fix.

Comment (cwu-okta) [inline _source/_docs/api/resources/oauth-clients.md]: According to the RFC (https://tools.ietf.org/html/rfc7591#section-2) this is a "server SHOULD display during consent." It is not a must.  Can we reword to indicate that this is up to the implementer to use if he chooses to do so? Okta will not necessarily use this yet (especially as we are still working through our consent spec.)

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth-clients.md]: Is this the only property that requires the user to do extra work? And who can give me the details around "necessarily"? i.e. does it ever work just by specifying a value. Just trying to figure out what to say.

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth-clients.md]: Where does the developer go to learn how to use this? What exactly do they have to implement?

Comment (cwu-okta) [inline _source/_docs/api/resources/oauth-clients.md]: Spoke to Mysti offline and let her know that Okta does not currently use this attribute.

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth-clients.md]: Will leave the description, add "not yet implemented in Okta."

**Pull request merged**